app.js

```diff
@@ -16,7 +15,7 @@ if (argv.data) {
   // that don't need a directory." But create it if it's
   // somehow missing (Mac for instance).
   if (!fs.existsSync('/var/lib/misc')) {
-    fs.mkdirSync('/var/lib/misc', 0700);
+    fs.mkdirSync('/var/lib/misc', 700);
```

Comment:
This was an intentional use of an octal constant.

If we're going to change to the ES6 version then it needs to be:

0o700

Just changing it to 700 would give you a different thing.

I just did some testing and the "new" syntax works back to node 4 (looks like that's when it came in). That's plenty old enough so please do it that way, with the `0o` prefix.